carbon-design-system/carbon
# refactor(react): add FileUploader and Slider behind wrapper in v11

Reference #10281 

This PR adds a helper function, `createClassWrapper`, that decorates class-based components behind the `enable-v11-release` feature flag. This only wraps `FileUploader` and `Slider`. The remaining components will be converted in separate PRs.

#### Changelog

**New**

- Add `createClassWrapper` helper function

**Changed**

- Update `FileUploader` entrypoint
- Update `Slider` entrypoint

**Removed**

#### Testing / Reviewing

- Verify that in `packages/react` the storybook **does not** have a wrapper show up in dev tools
- Verify that in `packages/carbon-react` the storybook **does** have a wrapper that shows up, for example

![Screen Shot 2022-01-06 at 12 16 28](https://user-images.githubusercontent.com/3901764/148431395-891f9a56-c904-4836-afe1-b1381c19dcbd.png)
